### PR TITLE
8289471: Issue in Initialization of keys in ErrorMsg.java and XPATHErrorResources.java

### PR DESCRIPTION
**Pull request by shruacha1234**
OpenJDK PR : https://github.com/openjdk/jdk/pull/9369
OpenJDK bug : https://bugs.openjdk.org/browse/JDK-8289471

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8289471](https://bugs.openjdk.org/browse/JDK-8289471): Issue in Initialization of keys in ErrorMsg.java and XPATHErrorResources.java

### Reviewers
 * [Paul Hohensee](https://openjdk.org/census#phh) (@phohensee - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk17u-dev pull/598/head:pull/598` \
`$ git checkout pull/598`

Update a local copy of the PR: \
`$ git checkout pull/598` \
`$ git pull https://git.openjdk.org/jdk17u-dev pull/598/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 598`

View PR using the GUI difftool: \
`$ git pr show -t 598`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk17u-dev/pull/598.diff">https://git.openjdk.org/jdk17u-dev/pull/598.diff</a>

</details>

### CODE REVIEW

**Comment by shruacha1234** [comment]
@RealCLanger Can you please review this PR

**Review by phohensee — APPROVED** [review]
Looks clean, lgtm.

**Comment by phohensee** [comment]
Pre-submit test failure looks like an infrastructure issue.

**Comment by phohensee** [comment]
Shruthi, do you need a spnsor?

**Comment by shruacha1234** [comment]
> Shruthi, do you need a spnsor?

Yes Paul

**Comment by RealCLanger** [comment]
> /integrate

Before integration, this needs maintainer approval. See point 6 of the [Wiki](https://wiki.openjdk.org/display/JDKUpdates/How+to+contribute+a+fix). Maybe Paul can help you to set the request label/comment.

**Comment by shruacha1234** [comment]
> > /integrate
> 
> Before integration, this needs maintainer approval. See point 6 of the [Wiki](https://wiki.openjdk.org/display/JDKUpdates/How+to+contribute+a+fix). Maybe Paul can help you to set the request label/comment.

OK

**Comment by phohensee** [comment]
Pre-submit test failure looks like an infrastructure issue.

**Comment by phohensee** [comment]
Tagged and commented on the JBS issue.

**Pull request closed without merging**
